### PR TITLE
Search bars are done

### PR DESCRIPTION
**Pull request by sancho325**
- [DEMO LINK](https://sancho325.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://sancho325.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,42 @@
...
+        class="formStyle"
+        method="post"
+        data-qa="small"
+      >
+        <img
+          src="/images/Search.svg"
+          alt="1"
+          class="imgStyle-2"
+        >
+        <input
```
Comment (Olexandr6) [inline src/index.html:44]:
```suggestion
        >
        
        <input
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,42 @@
...
+        class="formStyle"
+        data-qa="big"
+        method="post"
+      >
+        <img
+          src="/images/Search.svg"
+          alt="1"
+          class="imgStyle-1"
+        >
+        <input
```
Comment (Olexandr6) [inline src/index.html:25]:
```suggestion
        >
        
        <input
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin-top: 20px;
```
Comment (Olexandr6) [inline src/style.css:7]: Set this margin for the big form, not for body

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  border-radius: 3px;
+}
+
+input.inputStyle-2 {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```
Comment (Olexandr6) [inline src/style.css:78]: Don't duplicate styles, check the 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+input.inputStyle-2:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
```
Comment (Olexandr6) [inline src/style.css:93]: Don't duplicate styles, check the 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
```
Comment (mykola-koval) [inline src/style.css:5]: i think somewhere there should be the second @font-face

**Review by mykola-koval — CHANGES_REQUESTED** [review]
checkout my and Olexandr's comments 👇

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
You haven't fixed issues, particularly with duplicating styles, instead try to unify them

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  border-radius: 3px;
+  font-weight: 400;
+}
+
+.searchSmall:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: 400;
+}
```
Comment (jstmpelowycz) [inline src/style.css:79]: Duplicating styles, fix it

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url("/fonts/Avenir-Heavy.ttf");
+}
```
Comment (jstmpelowycz) [inline src/style.css:9]:
```suggestion
@font-face {
  font-family: "Avenir";
  src: url("/fonts/Avenir-Book.ttf");
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url("/fonts/Avenir-Heavy.ttf");
  font-weight: 900;
}
```

That's how it should look like

File: src/index.html [hunk 9]
```diff
@@ -11,6 +11,50 @@
...
+      >
+
+        <img
+          src="images/Search.svg"
+          alt="Search icon"
+          class="icon-1"
+        >
+
+        <input
+          class="style searchBig"
```
Comment (jstmpelowycz) [inline src/index.html:29]: Bad class name `style`

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
You still don't have text in your inputs bold on focus, fix it
Also, eliminate duplicating styles

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**